src/wallet/walletutil.cpp

```diff
@@ -49,6 +49,16 @@ static bool IsBerkeleyBtree(const fs::path& path)
     return data == 0x00053162 || data == 0x62310500;
 }
 
+static fs::path GetRelativePath(const fs::path& path, const fs::path& base)
+{
+    fs::path relative, parent = path;
+    while (!parent.empty() && parent != base) {
+      relative = parent.filename() / relative;
+      parent = parent.parent_path();
+    }
+    return relative;
```

Comment:
please add a unit test for this function

Reply:
Done.